test/integration/framework/framework.go

```diff
@@ -256,6 +257,15 @@ func ExpectWorkloadsToBeFrozen(ctx context.Context, k8sClient client.Client, cq
 	}, Timeout, Interval).Should(gomega.Equal(len(wls)), "Not enough workloads are frozen")
 }
 
+func ExpectWorkloadToBeAdmittedAs(ctx context.Context, k8sClient client.Client, wlNsName types.NamespacedName,
+	admission *kueue.Admission) {
```

Comment:
```suggestion
func ExpectWorkloadToBeAdmittedAs(ctx context.Context, k8sClient client.Client, wlNsName types.NamespacedName, admission *kueue.Admission) {
```

Reply:
Also, any reason not to accept a `kueue.Workload` like `ExpectWorkloadToBeAdmitted`?

Reply:
Added *kueue.Workload as function argument. Currently the func derives key from object only.
Earlier we were passing NamespacedName key.